### ISSUE TITLE
Validation is passing when it shouldn't

### ISSUE THREAD

**Issue by DaveyJH** [validator/validator#1310]
URL being validated or code to reproduce error:

```html
<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <title>Example</title>
</head>
<body>
  <h1>Multiple H1s</h1>
  <h1>Multiple H1s</h1>
  <h1>Multiple H1s</h1>
  <p>
    Paragraph with no close
    <ul>
      <li>List with no close
      <li></li>
      <li></li>
    </ul>
</body>
</html>
```

**Document checking completed. No errors or warnings to show.**

![image](https://user-images.githubusercontent.com/80708826/151258118-562fa4a9-e752-491f-a619-74c9d3f0c50a.png)

This has been tested on:
- Chrome
  - Mac
  - Windows 10
  - Android
- Firefox
  - Windows 10
- Edge
  - Windows 10

**Comment by cvrebert** [comment]
See https://github.com/validator/validator/issues/919#issuecomment-593650232
The spec sections for `<p>` and `<li>` likewise make closing tags optional for them.

**Comment by DaveyJH** [comment]
That's really interesting, thanks for the info. The example given is a much simpler version of a site I did not expect to pass validation due to similar errors. Similarly, am I right to assume the multiple `h1` elements are not an error and are just ill-advised? I would have expected at least a warning for some of the above. Apologies if I have misinterpreted multiple specs here.
The warnings and errors reported seemed very lacking on a few tests run but I am quite open to that being my own lack of experience and oversights.

**Comment by DaveyJH** [comment]
Having read into this more, is it possible to have this labelled as `not a bug`? It all appears to be technically correct with the specs. Very happy for the issue to be closed, not sure of best practice regarding that. @cvrebert any input appreciated 🙂
